### PR TITLE
fixed auth.jl

### PR DESCRIPTION
**Pull request by kwehmeyer**
Updated the authorization file with a comprehensive function that takes `client_id` and `client_secret` and returns a `SpotifyCredentials` object with all the necessary authentication information.

### CODE REVIEW

**Review by kwehmeyer — COMMENTED** [review]
great work me!

File: authorization.jl [hunk 1]
```diff
@@ -37,16 +37,16 @@ end
 
 function AuthorizeSpotify(client_id, client_secret)
     sc = SpotifyCredentials(client_id = client_id, client_secret = client_secret)
+    sc = base64encode(sc)
```
Comment (kwehmeyer) [inline authorization.jl:40]: ?

Comment (kwehmeyer) [reply authorization.jl:40]: what?

File: authorization.jl [hunk 2]
```diff
@@ -37,16 +37,16 @@ end
...
+    sc = GetAuthorizationToken(sc)
     return sc
 end
 
+#=
```
Comment (kwehmeyer) [inline authorization.jl:45]: What is #=

Comment (OwenDostie) [reply authorization.jl:45]: not sure, decent code but using is typed out in a not so nice to look at way

**Pull request merged**